package.json

```diff
@@ -29,7 +29,7 @@
   },
   "peerDependencies": {
     "karma": ">=0.9",
-    "requirejs": "~2.1"
+    "requirejs": "2.1.x | 2.2.x"
```

Comment:
May be just
```js
"requirejs": "^2.1.0"
```
Otherwise, we will have the same issue after requirejs 2.3 release